src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataBlockService.cs

```diff
@@ -66,6 +66,8 @@ public class DataBlockService : IDataBlockService
                 {
                     var dataBlock = _mapper.Map<DataBlock>(dataBlockCreate);
                     dataBlock.Created = DateTime.UtcNow;
+                    dataBlock.HighlightName = dataBlock.HighlightName?.Trim();
+                    dataBlock.HighlightDescription = dataBlock.HighlightDescription?.Trim();
```

Comment:
I don't think we do this kind of input sanitisation anywhere else on the backend. We leave it to the frontend, which this PR is also doing in `DataBlockDetailsForm.tsx`.

I think it's a good idea for the backend to have sanitisation but don't want this to set a precedent that we should be adding `.Trim()` everywhere. We'd litter the code with `.Trim()` and burden ourselves with spotting places where it's missing in new PR reviews.

Rather we should be coming up with a way to apply sanitisation on the backend globally for all requests with some kind of model binding, also with a way of annotating an exception cases if we think that's going to be required.

Reply:
As discussed on Slack, I'll add some details to the EES-4279 I've already created so it's clear we're looking for a complete solution to validation on the backend.

Also removed the `.Trims` I've added on the backend.